felix/wireguard/metrics_linux.go

```diff
@@ -12,6 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//go:build linux
```

Comment:
I believe we only need the `go:build linux` line here, since we build with a well-known golang version and the `+build` syntax is an older version of the same instruction.